exec.go

```diff
@@ -223,9 +223,9 @@ func execute(t *Text, aq0 int, aq1 int, external bool, argt *Text) {
 			}
 		}
 		if a != "" {
-			t.w.Eventf("%c0 0 0 %d %v\n", c, len(a), a)
+			t.w.Eventf("%c0 0 0 %d %v\n", c, len([]rune(a)), a)
 			if aa != "" {
-				t.w.Eventf("%c0 0 0 %d %v\n", c, len(aa), aa)
+				t.w.Eventf("%c0 0 0 %d %v\n", c, len([]rune(aa)), aa)
```

Comment:
And now I'm not sure about this one.  I'd *like* the interface across the channel to be counted in runes, not bytes. But I have only the evidence of win's implementation, and of Russ's acme port that this is the case.
We need tests.

Reply:
Ok, carefully re-reading the man page, russ's go code, the original acme code, I believe that yes, the "number of characters" refers to runes, not bytes.  But the rune counts change should be a separate CL.  I'll split it off.